src/cassy.py

```diff
@@ -205,6 +206,37 @@ def get_user_password(username):
     except Exception as e:
         raise Exception('Transaction Error Occurred: ' + str(e))
 
+def get_user_email(username):
+    """
+    Obtains email for the requested user.
+    """
+
+    values = {}
+    values['username'] = username
+    auth_stmt_get = session.prepare(
+        'SELECT email'
+        + ' FROM ' + os.environ.get('DB_USER_TABLE')
+        + ' WHERE username=?'
+    )
+    bound = auth_stmt_get.bind(values)
+    session.row_factory = named_tuple_factory
+
+    try:
+        if username == '':
+            raise PlantalyticsLoginException(EMAIL_ERROR)
```

Comment:
The EMAIL_ERROR doesn't necessarily seem like a LoginException. Although there is no functional difference between our PlantalyticExceptions, they're there for book-keeping purposes so it might be useful to create a new one. Or not.